cmd/handler-api.go

```diff
@@ -150,12 +151,17 @@ func (t *apiConfig) init(cfg api.Config, setDriveCounts []int) {
 	t.staleUploadsExpiry = cfg.StaleUploadsExpiry
 	t.staleUploadsCleanupInterval = cfg.StaleUploadsCleanupInterval
 	t.deleteCleanupInterval = cfg.DeleteCleanupInterval
+	t.disableODirect = cfg.DisableODirect
 }
 
-func (t *apiConfig) getListQuorum() int {
+func (t *apiConfig) isDisableODirect() bool {
 	t.mu.RLock()
 	defer t.mu.RUnlock()
 
+	return t.disableODirect
+}
+
+func (t *apiConfig) getListQuorum() int {
```

Comment:
Why the lock is removed here in getListQuorum() ?

Reply:
my bad